package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@gaus/update-types",
```

Comment:
is @ gaus actually available on npm? like no user already has that username?

Reply:
Fixed